### ISSUE TITLE
[java] AvoidLosingExceptionInformation generates a false positive

### ISSUE THREAD

**Issue by Noah0120** [pmd/pmd#3439]
**Affects PMD Version:** 
6.36.0
**Rule:**
AvoidLosingExceptionInformation
Please provide the rule name and a link to the rule documentation:
https://pmd.github.io/latest/pmd_rules_java_errorprone.html#avoidlosingexceptioninformation

**Description:**
When printing the information of an exception, 
it detected it as it loses the information of the exception.

**Code Sample demonstrating the issue:**

```
class AvoidLosingExceptionInformation{
        public void test(){
                try{
                      //...
                } catch (Exception e){
                        e.getMessage().printStackTrace(System.err);
                }
        }
}
```

**Expected outcome:**
It is a false positive. It should not give any detection results.
Thank you for your consideration.

**Running PMD through:** *[CLI]*

**Comment by oowekyala** [comment]
I don't think the sample you provided compiles... Exception#getMessage returns a string. Maybe you meant to type `e.printStackTrace(System.err);`, which should not trigger the rule. Please reopen if the problem occurs on code that compiles.
